Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+      .addParameter("expression", _("Opacity in %"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("setOpacity")
+      .setManipulatedType("number")
+      .setGetter("getOpacity");
+
+    object
+      .addCondition(
+        "GetOpacity",
+        _("Opacity"),
+        _("Test the opacity of an video object"),
+        _("Opacity of _PARAM0_ is _PARAM1_ _PARAM2_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("relationalOperator", _("Sign of the test"))
+      .addParameter("expression", _("Opacity 0-100"))
+      .getCodeExtraInformation()
+      .setFunctionName("getOpacity")
+      .setManipulatedType("number");
+
+    object
+      .addExpression(
+        "GetOpacity",
+        _("Get current opacity"),
+        _("Return the opacity of an video object"),
+        "",
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("getOpacity");
+
+    object
+      .addAction(
+        "SetPlaybackSpeed",
+        _("Set playback speed (in %)"),
+        _("Set playback speed (in %)"),
+        _("Do _PARAM1_ _PARAM2_ % to the playback speed of _PARAM0_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("operator", _("Modification's sign"), "", false)
+      .addParameter("expression", _("Speed in %"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("setPlaybackSpeed")
+      .setManipulatedType("number")
+      .setGetter("getPlaybackSpeed");
+
+    object
+      .addCondition(
+        "GetPlaybackSpeed",
+        _("Playback speed "),
+        _("Test the playback speed of an video object"),
+        _("Playback speed of _PARAM0_ is _PARAM1_ _PARAM2_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("relationalOperator", _("Sign of the test"))
+      .addParameter("expression", _("Speed 0-100"))
+      .getCodeExtraInformation()
+      .setFunctionName("getPlaybackSpeed")
+      .setManipulatedType("number");
+
+    object
+      .addExpression(
+        "GetPlaybackSpeed",
+        _("Get current playback speed"),
+        _("Return the playback speed of an video object"),
+        "",
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("getPlaybackSpeed");
+
+    /**
+     *
+     * //TODO Action for change video (with PATH expression fileSystem) of an object video
```

Comment:
I don't think this is a good idea, it won't work for browser games. For now, let's avoid changing anything, much like you can't modify an object animations.